h/static/scripts/controllers.coffee

```diff
@@ -22,13 +22,13 @@ class AppController
   this.$inject = [
     '$controller', '$document', '$location', '$route', '$scope', '$window',
     'auth', 'drafts', 'identity',
-    'permissions', 'streamer', 'annotationUI',
+    'permissions', 'streamer', 'annotationUI', 'host',
```

Comment:
We had a talk about this. I suggest putting that part into app.coffee where we configure the other module related services. This way it won't broke if we accidentally we not inject the service to anywhere else